defaults/main.yml

```diff
@@ -24,24 +29,27 @@ postgres_labels:
   kubernetes.io/cluster-service: "true"
 
 postgres_volumes:
+    # PGDATA volume
     - name: varlibpostgres
-      hostPath: 
-        path: /var/lib/postgres/data
+      hostPath:
+        path: /var/lib/postgresql/data
```

Comment:
hmm what if the rc runs 2 slaves on the same node.
I know it's a dumb idea for postgres but what about services which don't need to be bound to a specific node ?